rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py

```diff
@@ -2,6 +2,23 @@
 import numpy as np
 
 
+def cleanup_tokens(
+    token_ids_string: List[Tuple[int, Text]], delimiter: Text
+) -> Tuple[List[int], List[Text]]:
+    """Utility method to apply specific delimiter based cleanup on list of tokens"""
```

Comment:
We should use the full docstring symtax, e.g. including `Args` and `Return`. But we also have another issue for that.

Reply:
Thanks for pointing it out. Will address it as part of that issue.